submissions/busolaAdedoyin/contactFormPage/contactForm.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Login Page</title>
+</head>
+<body>
+<fieldset>
+    <legend>Login Form</legend>
+    <h1>Login Page</h1>
+    <form>
+        <label for="fullname"> Full Name</label><br>
+        <input type="text" name="firstname" id="firstname"><br>
+        <label for="email">Email</label><br>
+        <input type="email" name="email" id="email" placeholder="Enter your email address"><br>
+        <label for="subject">Subject</label><br>
+        <input type="text" name="subject" id="subject"><br>
+        <label for="messageinput">Message</label><br>
+        <textarea name="messageinput" id="messageinput" cols="30" rows="10"></textarea><br>
```

Comment:
Update the name attribute and id as indicated above

Reply:
I have made the changes